### PR TITLE
Ragdoll on death

### PR DESCRIPTION
**Pull request by Nowp**

### CODE REVIEW

**Comment by Nowp** [comment]
[Ragdoll on death](https://trello.com/c/spFE3r8g/54-ragdoll-on-death)

File: code/Player/MainPlayer.cs [hunk 1]
```diff
@@ -144,10 +153,31 @@ private void ServerControls()
...
+
         private void InitRole()
         {
             ClothesHandler.AttachClothes(RoleHandler.Role.Clothing);
         }
+
+        public override void TakeDamage(DamageInfo info)
+        {
+            base.TakeDamage(info);
+            Log.Info($"Health: {Health}");
```
Comment (Prakkkmak) [inline code/Player/MainPlayer.cs:165]:
```suggestion
```

File: code/Player/MainPlayer.cs [hunk 2]
```diff
@@ -1,38 +1,45 @@
...
-
             if (Host.IsServer)
             {
                 Inventory = new Inventory(MaxInventoryCapacity);
                 GaugeHandler = new GaugeHandler();
                 ClothesHandler = new ClothesHandler(this);
                 RoleHandler = new RoleHandler();            }
         }
-        
+
```
Comment (Prakkkmak) [inline code/Player/MainPlayer.cs:28]:
```suggestion
```

File: code/Entities/PlayerCorpse.cs [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+		{
+			PhysicsGroup.AddVelocity( force );
+
+			if (forceBone >= 0)
+			{
+				PhysicsBody body = GetBonePhysicsBody( forceBone );
+
+				if (body != null)
+				{
+					body.ApplyForce( force * 1000 );
```
Comment (Prakkkmak) [inline code/Entities/PlayerCorpse.cs:55]: magic number

File: code/Entities/PlayerCorpse.cs [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+			this.CopyBonesFrom(player);
+			this.SetRagdollVelocityFrom(player);
+
+			foreach (Entity child in player.Children)
+			{
+				if (child is ModelEntity e)
+				{
+					string model = e.GetModelName();
+
+					if (model != null && !model.Contains("clothes"))
```
Comment (Prakkkmak) [inline code/Entities/PlayerCorpse.cs:35]: magic string

**Review by Prakkkmak — CHANGES_REQUESTED** [review]

**Review by Prakkkmak — CHANGES_REQUESTED** [review]
pas fais

**Review by Prakkkmak — APPROVED** [review]

**Pull request merged**